### ISSUE TITLE
Assertions for datalists needed

### ISSUE THREAD

**Issue by Tray2** [sinnbeck/laravel-dom-assertions#7]
I think this package needs a AssertDatalist class, so that we can perform assertions on the datalist.

```php
$form->assertDatalistExists('#authors', function(AssertDatalist $datalist) {
   $datalist->hasValue('Jordan, Robert');
   $datalist->hasValues([
      'Eddings, David',
      'Jordan, Robert',
    ]);
});  
```

**Comment by sinnbeck** [comment]
I'll look into that. But isn't datalists single value as they are bound to an input? So only `->hasValue()` would make sense here

**Comment by Tray2** [comment]
They have multiple options just like a select would.

```php
    <label for="format">Format:</label>
    <input list="formats" value="{{ old('format_name', $book->format) }}" id="format" name="format_name" placeholder="Format...">
    <datalist id="formats">
        @foreach($formats as $format)
            <option value="{{ $format->name }}"></option>
        @endforeach
    </datalist>
```

**Comment by Tray2** [comment]
I think maybe that this syntax might be more appropiate since it has options rather than values, even though the option uses the attribute value.

```php
it('contains a list of authors', function () {
    get(route('books.edit', $this->book))
        ->assertOk()
        ->assertFormExists(function (AssertForm $form){
            $form->findDatalist('#authors', function (AssertDataList $datalist) {
                $datalist->hasOptions([
                    'Eddings, David',
                    'Goodkind, Terry'
                ]);
           });
        });
});
```

**Comment by sinnbeck** [comment]
It has been released in 1.1.1
